google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/prepare/sql_objects/sql_objects_datacatalog_entry_factory_test.py

```diff
@@ -84,7 +82,7 @@ def test_make_entry_for_stored_procedure_sql_object_should_set_all_available_fie
             utils.Utils.convert_json_to_object(self.__MODULE_PATH,
                                                'sql_objects_config.json')
 
-        factory = sql_objects_datacatalog_entry_factory.\
+        factory = sql_objects.sql_objects_datacatalog_entry_factory.\
```

Comment:
```suggestion
        factory = sql_objects.SQLObjectsDataCatalogEntryFactory(
```
Please replace all similar occurrences to turn the code less verbose if it makes sense.